reference/import_export/product-import.rst

```diff
@@ -160,87 +188,61 @@ We provide here specific implementations for these elements, the services are de
+               - '@pim_connector.array_converter.flat_to_standard.product_delocalized'
+               - '@pim_connector.reader.file.media_path_transformer'
 
 .. note::
 
-    This step is able to extract a Zip archive which contains a CSV file and a folder for related images or files. The CSV file has to use relative paths to reference the files.
-
-Product Processor
------------------
-
-This element receives items one by one, fetches or creates the related product, updates and validates it.
-
-The service is defined in ``src\Pim\Bundle\ConnectorBundle\Resources\config\processors.yml``.
-
-.. code-block:: yaml
-
-    parameters:
-        pim_connector.processor.denormalization.product.class: Pim\Component\Connector\Processor\Denormalization\ProductProcessor
-
-    services:
-        pim_connector.processor.denormalization.product.flat:
-            class: %pim_connector.processor.denormalization.product.class%
-            arguments:
-                - '@pim_connector.array_converter.flat.product'
-                - '@pim_catalog.repository.product'
-                - '@pim_catalog.builder.product'
-                - '@pim_catalog.updater.product'
-                - '@pim_catalog.validator.product'
-                - '@akeneo_storage_utils.doctrine.object_detacher'
-                - '@pim_catalog.comparator.filter.product'
-                - '@pim_catalog.localization.localizer.converter'
-
-The class ``Pim\Component\Connector\Processor\Denormalization\ProductProcessor`` mainly delegates the operations to different technical and business services.
-
-.. code-block:: php
-
-    /**
-     * @param StandardArrayConverterInterface       $arrayConverter array converter
-     * @param IdentifiableObjectRepositoryInterface $repository     product repository
-     * @param ProductBuilderInterface               $builder        product builder
-     * @param ObjectUpdaterInterface                $updater        product updater
-     * @param ValidatorInterface                    $validator      product validator
-     * @param ObjectDetacherInterface               $detacher       detacher to remove it from UOW when skip
-     * @param ProductFilterInterface                $productFilter  product filter
-     * @param AttributeLocalizedConverterInterface  $localizedConverter attributes localized converter
-     */
-    public function __construct(
-        StandardArrayConverterInterface $arrayConverter,
-        IdentifiableObjectRepositoryInterface $repository,
-        ProductBuilderInterface $builder,
-        ObjectUpdaterInterface $updater,
-        ValidatorInterface $validator,
-        ObjectDetacherInterface $detacher,
-        ProductFilterInterface $productFilter,
-        AttributeLocalizedConverterInterface $localizedConverter
-    ) {
-        // ...
-    }
+    This step is able to extract a Zip archive which contains a file and a folder for related images or files. The file has to use relative paths to reference the files.
```

Comment:
which contain a CSV file for products and next to it a folder containing images. The product file refers to images using relatives paths.